apps/example/src/app/RudderEvents.tsx

```diff
@@ -101,7 +101,16 @@ const RudderEvents = () => {
   };
 
   const alias = () => {
-    rudderClient.alias('RN Alias ID', 'previous RN userId');
+    rudderClient.alias('New User ID');
+
+    // All the below calls are also valid usages of alias
+    // Anything else is invalid
+    // rudderClient.alias('New User ID', null);
+    // rudderClient.alias('New User ID', {});
+    // rudderClient.alias('New User ID', undefined);
+
```

Comment:
Let's add the last case which highlights the steps to pass the RudderOptions object.
```suggestion
    // rudderClient.alias("New User ID", getLocalOptions());

```

Reply:
Done.